OrangeDesignSystemDemo/OrangeDesignSystemDemoTests/Data/AppNewsMock_notCompliant.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "version": "0.15.0",
```

Comment:
D'escrime somewhere the Walid sytae because i'am share the syntaxe will be open wrong.

Reply:
Yep I agree. I will look to JSON schema and update the doc :)